services/shuttle-actix-web/README.md

```diff
@@ -0,0 +1,23 @@
+## Shuttle service integration for the Actix Web framework.
+
+### Example
+
+````rust
+ use actix_web::{get, web::ServiceConfig};
+ use shuttle_actix_web::ShuttleActixWeb;
+
+ #[get("/")]
+ async fn hello_world() -> &'static str {
+     "Hello World!"
+ }
+
+ #[shuttle_runtime::main]
+ async fn actix_web() -> ShuttleActixWeb<impl FnOnce(&mut ServiceConfig) + Send + Clone + 'static> {
+     let config = move |cfg: &mut ServiceConfig| {
+         cfg.service(hello_world);
+     };
+
+     Ok(config.into())
+ }
+ 
+````
```

Comment:
The codeblock should have 3 backticks, and not be indented with 1 space, and I think we still need `,no_run` to not trigger them in doc tests.